app/scenes/ApiKeyNew/components/ExpiryDatePicker.tsx

```diff
@@ -0,0 +1,116 @@
+import { faCalendar } from "@fortawesome/free-solid-svg-icons";
```

Comment:
```suggestion
import { CalendarIcon } from "outline-icons";
```

Please use `outline-icons` for UI, only using FA for the user-selectable icons. If new icons are needed that aren't in `outline-icons` we'll make them.

Reply:
Ah! This is missing in the [IconLibrary](https://github.com/outline/outline/blob/main/shared/utils/IconLibrary.tsx#L220).
It'd be nice to add the missing icons so they can be used as collection/document icons.

Reply:
Yes, not all available icons are in there as many don't make sense – Calendar could be though